superset-frontend/src/dashboard/actions/sliceEntities.js

```diff
@@ -39,92 +39,154 @@ export function fetchAllSlicesFailed(error) {
   return { type: FETCH_ALL_SLICES_FAILED, payload: { error } };
 }
 
+function fetchSlices(
+  userId,
+  excludeFilterBox,
+  dispatch,
+  filter_value,
+  sortColumn = 'changed_on_delta_humanized',
+  slices = {}
```

Comment:
do you need the current slices? aren't we fetching here and need to replace the existing ones?